python/GafferUITest/ViewTest.py

```diff
@@ -81,10 +81,6 @@ def testEditScope( self ) :
 		editScope = Gaffer.EditScope()
 		editScope.setup( view["in"] )
 
-		self.assertFalse( view["editScope"].acceptsInput( addNode["sum"] ) )
-		self.assertFalse( view["editScope"].acceptsInput( editScope["in"] ) )
-		self.assertTrue( view["editScope"].acceptsInput( editScope["out"] ) )
-
```

Comment:
Is the implementation of `acceptsInput` coming back? Seemed like a good idea.

Reply:
No, as explained in the commit message, it's not viable. In practice its not really a big deal either - the plug is only exposed via EditScopePlugValueWidget, and that maintains the constraint for us.

Reply:
Ahh sorry, I'd missed that as the email link had scrolled that off screen! Sorry!